### ISSUE TITLE
Update Documentation to reflect steps on how to update guides.dataverse.org at release time

### ISSUE THREAD

**Issue by poikilotherm** [IQSS/dataverse#6295]
When I access http://guides.dataverse.org, I see the following index:
![latest](https://user-images.githubusercontent.com/5468333/67187271-229b2300-f3ea-11e9-95d9-e86d26496a25.png)

Switching to "4.17" results in the same:
![4.17](https://user-images.githubusercontent.com/5468333/67187534-b5d45880-f3ea-11e9-850d-0855087522f4.png)

Switching to "4.16" gives:
![4.16](https://user-images.githubusercontent.com/5468333/67187335-43fc0f00-f3ea-11e9-9ad0-75d464cc3e54.png)

Building the docs locally uses proper "4.17" as version string. Maybe a build time caching issue?
![local](https://user-images.githubusercontent.com/5468333/67187606-d4d2ea80-f3ea-11e9-9736-78846925a394.png)

**Comment by djbrooke** [comment]
Thanks @poikilotherm, we'll investigate.

**Comment by djbrooke** [comment]
This has been fixed for 4.17. I'll use this issue to update the documentation for the release process concerning the docs.

**Comment by djbrooke** [comment]
As previously mentioned, this has been fixed for the 4.17 guides. We did something out of the usual order during our release process. We'll take our usual steps for 4.18, and if it works as expected I'll use this issue to document the steps. If it does not work as expected, we'll investigate, fix, and document the correct process. Either way, I'll let this sit until we go through the process again (hopefully soon for 4.18 :))
